release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -132,6 +132,14 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
 [id="ocp-4-10-web-console"]
 === Web console
 
+[id="ocp-4-10-dynamic-plugin-technology-preview"]
+==== Dynamic Plug-in (Technology Preview)
+Starting with {product-title} 4.10, OpenShift console dynamic plug-ins are now available as a Technology Preview feature. This feature allows you make customizations to your interface at runtime such as: add custom pages, add perspectives and update navigation items, and add tabs and actions to resource pages. Plug-ins are registered with the console using the `ConsolePlugin` custom resource, and they are enabled in the console operator configuration by a cluster administrator.
+
+[id="ocp-4-10-multicluster-console-technology-preview"]
+==== Multi-cluster Console (Technology Preview)
+Starting with {product-title} 4.10, the multi-cluster console is now available as a Technology Preview feature. This feature supports a single {product-title} console that can connect to remote clusters API. An *All clusters* perspective contains the Advanced Cluster Management(ACM) multi-cluster views. This feature also implements single sign-on(SSO), so you can bypass the login prompt when navigating between clusters.
```

Comment:
We'll need to document how to enable the multi-cluster feature somewhere. It requires setting a feature gate.

The "All clusters" perspective will only be available if you have ACM 2.5 installed and enable the ACM console plugin, which is a manual step an admin needs to take.

I'd remove or rephrase the sentence about SSO. We have an IDP config operator that will push configuration for either GitHub or LDAP identity providers to all clusters in the fleet, but it's a separate operator to install and configure.

cc @alimobrem @showeimer @jamestalton @robinbobbitt